octopoes/octopoes/models/ooi/scans.py

```diff
@@ -0,0 +1,17 @@
+from typing import Literal
+
+from octopoes.models import OOI, Reference
+
+
+class Scan(OOI):
```

Comment:
I was a bit confused about this (new) object type, so perhaps we could add a docstring here to elaborate? Is it a generic wrapper for types that are not available yet, and should it in that case be called "AbstractOOI" or "GenericOOI" or something? Or if it is mainly a way to pin raw files for garbage collection, should we perhaps call this "RawFileSource"?

Reply:
I agree it would be good to add a docstring.

It is an object that represents a scan done outside of OpenKAT. This can for example be a scan with burpsuite which we can't do in a boefje. The user would be doing multiple scans on different thing so we need to a was to link the uploaded raw files and generated OOIs with the different scans. Sometimes there will already be an OOI to link the scan to, for example a network or IP address. This might not always be the case, so that's why we introduced this object. See the linked issues for more context/discussion about it.

Maybe something like "ExternalScan" might be a better name.

Reply:
Renamed to ExternalScan